Index: run_example.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from engine.awsUploader import awsUploader\r\nfrom engine.aws_upload_config import aws_config\r\nimport boto3\r\n\r\ndef main():\r\n    uploader = awsUploader()\r\n\r\n    # מגדירים פרטי התחברות ל־MySQL\r\n    uploader.set_mysql_connection_details(\"mysql-source-db2.cdg0qswm8uxu.us-east-1.rds.amazonaws.com\", aws_config[\"source\"])\r\n\r\n    uploader.connect()\r\n\r\n    # ⬆\uFE0F מעלים את הסכמה והדאטה (שני הקבצים)\r\n    uploader.upload(\"sakila-schema.sql\")\r\n    uploader.upload(\"sakila-data.sql\")\r\n\r\nif __name__ == \"__main__\":\r\n    main()\r\n\r\n'''\r\nrds = boto3.client(\"rds\")\r\n    instance_id = aws_config[\"source\"][\"DBInstanceIdentifier\"]\r\n    response = rds.describe_db_instances(DBInstanceIdentifier=instance_id)\r\n    endpoint = response[\"DBInstances\"][0][\"Endpoint\"][\"Address\"]\r\n    print(\"\uD83D\uDD0D Endpoint:\", endpoint)\r\n'''
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/run_example.py b/run_example.py
--- a/run_example.py	(revision 9606f59e8b3c28a73c7a88387dd549d203dafe8a)
+++ b/run_example.py	(date 1744114675608)
@@ -23,4 +23,5 @@
     response = rds.describe_db_instances(DBInstanceIdentifier=instance_id)
     endpoint = response["DBInstances"][0]["Endpoint"]["Address"]
     print("🔍 Endpoint:", endpoint)
-'''
\ No newline at end of file
+'''
+
